### PR TITLE
feat: Updates documentation after switching from fiveg_gnb_identity to fiveg_core_gnb

### PR DESCRIPTION
**Pull request by Gmerold**
# Description

- Replaces `fiveg_gnb_identity` with `fiveg_core_gnb`
- Removes references to `v1.5` release branch
- Fixes `not a snap cgroup` issue when accessing lxc machines
- Bumps version of the `terraform-lxd` provider to the latest one

# Checklist:

- [ ] Documentation follows the [Canonical Documentation Style Guide](https://docs.ubuntu.com/styleguide/en)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

File: docs/tutorials/getting_started.md [hunk 1]
```diff
@@ -167,42 +167,43 @@ Model   Controller          Cloud/Region        Version  SLA          Timestamp
...
+grafana-agent/0*             blocked   idle   10.1.194.196         Missing ['grafana-cloud-config']|['logging-consumer'] for logging-provider; ['grafana-cloud-config']|['send-remote-wr...
+mongodb/0*                   active    idle   10.1.194.249         
+nms/0*                       active    idle   10.1.194.203         
+nrf/0*                       active    idle   10.1.194.247         
+nssf/0*                      active    idle   10.1.194.227         
+pcf/0*                       active    idle   10.1.194.224         
+router/0*                    active    idle   10.1.194.245         
+self-signed-certificates/0*  active    idle   10.1.194.225         
+smf/0*                       active    idle   10.1.194.255         
+traefik/0*                   error     idle   10.1.194.246         hook failed: "certificates-relation-changed"
```
Comment (gatici) [inline docs/tutorials/getting_started.md:198]: In the first deployment, Traefik fails with ` hook failed: "ingress-relation-created"` not `certificates-relation-changed`.

Comment (Gmerold) [reply docs/tutorials/getting_started.md:198]: You mean the `ingress-relation-created` looks better? :) 
From the functional perspective it doesn't matter on which event does the Traefik fail. In both cases the reason is the same - the LB IP address is not a valid URL.

File: docs/tutorials/getting_started.md [hunk 2]
```diff
@@ -333,15 +328,13 @@ ran    microk8s-localhost  microk8s/localhost  3.6.0    unsupported  12:18:26+02
 
 SAAS  Status  Store  URL
 amf   active  local  admin/sdcore.amf
+nms   active  local  admin/sdcore.nms
 
-App     Version  Status  Scale  Charm              Channel     Rev  Address         Exposed  Message
-gnbsim  1.5.0    active      1  sdcore-gnbsim-k8s  1.5/stable  612  10.152.183.209  no
+App     Version  Status   Scale  Charm              Channel   Rev  Address        Exposed  Message
+gnbsim  1.4.5    waiting      1  sdcore-gnbsim-k8s  1.6/edge  638  10.152.183.85  no       installing agent
```
[the remark this answers is not in the document]
Comment (Gmerold) [reply docs/tutorials/getting_started.md:334]: That's actually correct. `Waiting for TAC and PLMNs configuration` is a unit message not the app message.

[the remark this answers is not in the document]
Comment (gatici) [reply docs/tutorials/getting_started.md:334]: image

This is what appears.

[the remark this answers is not in the document]
Comment (Gmerold) [reply docs/tutorials/getting_started.md:334]: Interesting :D Here's what I see:

image

[the remark this answers is not in the document]
Comment (gatici) [reply docs/tutorials/getting_started.md:334]: Not a  big issue.

**Review by gatici — APPROVED** [review]

**Review by patriciareinoso — APPROVED** [review]

**Pull request merged**